src/cloudant/document.py

```diff
@@ -17,13 +17,19 @@
 """
 import json
 import posixpath
-import urllib
 import requests
-
 from requests.exceptions import HTTPError
 
+# pylint: disable=wrong-import-order
```

Comment:
given this stanza appears a couple of times and I think string quoting is going to be used in other places, may be cleaner to set up a quote_plus_ style symbol in the _2to3 module.